### PR TITLE
Thought Addons 1#

### PR DESCRIPTION
**Pull request by Clumxy**
Some new Thoughts. this does **not** mean that the Thought Overhaul is over.

### CODE REVIEW

**Review by keyraven — CHANGES_REQUESTED** [review]
Since this is new content, can you tag this with the correct pronoun tags? [Here is the guide. ](https://docs.google.com/document/d/1qcks1E25NkOfU6zdwUlccU42VRkHv_WNaCsMpUP5jDA/edit)

**Comment by Clumxy** [comment]
> 

Sorry, but I dont think i really have the time to do that. These were made before the pronouns were added. Im sorry and I hope you understand.

File: resources/dicts/thoughts/alive/kitten.json [hunk 1]
```diff
@@ -129,32 +175,18 @@
         "thoughts": [
             "Thinks they saw a StarClan cat in their dreams",
             "Is mesmerized by the night sky",
-            "Thinks they're the leader of their own Clan",
+            "Thinks theyre the leader of their own clan",
```
Comment (SunlitFable) [inline resources/dicts/thoughts/alive/kitten.json]: i pointed this out last time, but it still hasn't been fixed?

Comment (Clumxy) [reply resources/dicts/thoughts/alive/kitten.json]: I'll try to fix that.

**Pull request closed without merging**